### ISSUE TITLE
Memory leaks in UICollectionView binding.

### ISSUE THREAD

**Issue by nayzak** [ReactiveKit/Bond#486]
CollectionView cells that were build in `createCell` block are not deallocated.
```swift
class Cell: UICollectionViewCell {

  deinit {
    print("!!! Deinit: \(self)") // Not called
  }
}

class CollectionViewController: UICollectionViewController {

  init() {
    let layout = UICollectionViewFlowLayout()
    super.init(collectionViewLayout: layout)
  }

  required init?(coder aDecoder: NSCoder) {
    fatalError("init(coder:) has not been implemented")
  }

  deinit {
    print("!!! Deinit: \(self)") // Called
  }

  override func viewDidLoad() {
    super.viewDidLoad()

    self.collectionView!.register(Cell.self, forCellWithReuseIdentifier: "Cell")

    let dataSource: SafeSignal<[Int]> = SafeSignal.just(Array(0...10))

    dataSource.bind(to: self.collectionView!) { _, index, collection in
      return collection.dequeueReusableCell(withReuseIdentifier: "Cell", for: index)
    }
  }
}

```

**Comment by sblepa** [comment]
This also happens for UITableViewCell when binding the tableview

**Comment by srdanrasic** [comment]
Thanks everyone for reporting this. It should be fixed in v6.5.1.

**Comment by phjacobs7AG** [comment]
Has it been fixed yet ?
